shoplist/src/main/java/com/netguru/codereview/ui/MainViewModel.kt

```diff
@@ -0,0 +1,43 @@
+package com.netguru.codereview.ui
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.netguru.codereview.network.ShopListApiMock
+import com.netguru.codereview.network.ShopListRepository
+import com.netguru.codereview.network.model.ShopListItemResponse
+import com.netguru.codereview.network.model.ShopListResponse
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.launch
+
+class MainViewModel : ViewModel() {
```

Comment:
```suggestion
class MainViewModel(private val ioDispatcher: CoroutineDispatcher = Dispatchers.IO) : ViewModel() {
```

Make sure to inject your dispatchers to make easier to change them while doing unitTest or instrumentationTests.